test/types.test.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  JsonMLAttributes,
+  JsonMLChildren,
+  utils,
+} from '..';
+
+// Utils
+const isFragYes: boolean = utils.isFragment(['', 'hi']);
+const isFragNo: boolean = utils.isFragment(['div', 'hi']);
```

Comment:
Is there a better way to assert types?

Reply:
`suggestion`: You might like to take a look at using [assertion functions](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions). There are various options.

Reply:
`suggestion`: Another less obvious option would be to add a test script which exercises the types using the TypeScript compiler. Here's an example from [core-logbus-sdk](https://github.com/CondeNast-Copilot/core-logbus-sdk/blob/master/package.json#L24).

Reply:
I didn't make use of `assert` since (currently) this file is only run through `tsc` and not executed at runtime. Additionally, I guess I'd have to figure out how to specify assert/node types in the config?

I did look at other repo examples, and the logbus is another good reference
https://github.com/CondeNast-Copilot/core-logbus-sdk/blob/master/test/assert-types.ts

It feels somewhat similar to what I'm doing - which is essentially just trying to setup all the calls and objects. I went the extra difference of assigning the return value so the type checker would catch this also.